src/components/message.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/components/message.scss';
+import WarningIcon from '../svg/warning.svg';
+import CloseIcon from '../svg/times.svg';
+
+const MessageLevel = {
+  warning: 'warning',
+  failure: 'failure',
+  success: 'success',
+  info: 'info',
+};
+
+const iconSize = '1.125em';
+
+const Message = ({ children, level, onDismiss }) => (
+  <section className={`alert alert--${level}`}>
+    <section className="alert__content">
+      <WarningIcon width={iconSize} height={iconSize} />
+      <small>{children}</small>
+    </section>
+    {onDismiss && (
+      <button type="button" className="alert__dismiss" onClick={onDismiss}>
+        <CloseIcon width="10" height="10" />
+      </button>
+    )}
+  </section>
+);
+
+Message.propTypes = {
+  /**
+   * The message body
+   */
+  children: PropTypes.element.isRequired,
```

Comment:
`children` could also be `PropTypes.string`

Reply:
`Proptype.node` then?

Reply:
👍